changelogs/fragments/1927-job_update-system-check.yml

```diff
@@ -0,0 +1,4 @@
+trivial:
+   - module_utils/job.py - In some systems, system and subsystem return values from jls are returned as None values,
+     breaking the current implementation. This ensures that the job modules will not fail even under that circumstances.
```

Comment:
last word (circumstances) should be singular (circumstance).

Reply:
Thank you Rich. Updated it :)